video/out/wayland_common.c

```diff
@@ -2098,6 +2240,25 @@ static int handle_round(int scale, int n)
     return (scale * n + WAYLAND_SCALE_FACTOR / 2) / WAYLAND_SCALE_FACTOR;
 }
 
+static void initialize_color_maps(struct vo_wayland_state *wl)
+{
+    // -1 indicates unsupported
```

Comment:
Any reason to use -1 for unsupported instead of 0?

Reply:
Because 0 is a real value in the enums for the protocol and compositors might not support it.